File: Practice/atcoder/ABC/139/src/c.py
# 逆に考えるんだ
n = int(input())
arr = list(reversed(list(map(int, input().split()))))
ans = [0 for _ in range(n)]
for i in range(1, n):
    if arr[i - 1] <= arr[i]:
        ans[i] = ans[i - 1] + 1

print(max(ans))

File: Practice/atcoder/ABC/139/src/c.php
<?php
  $n = intval(trim(fgets(STDIN)));
  $arr = array_map("intVal", explode(" ", trim(fgets(STDIN))));
  $arr = array_reverse($arr);
  $ans = array_fill(0, $n, 0);
  for($i=1; $i<$n; $i++){
    if($arr[$i] >= $arr[$i-1]){
      $ans[$i] = $ans[$i-1] + 1;
    }
  }
  echo max($ans) . PHP_EOL;
?>